### DISCUSSION TITLE
Bare-Metal Version

### DISCUSSION THREAD

**Discussion by rabejens** [BlitterStudio/amiberry#739] [Ideas]
With the Pi400 being the perfect modern and compact “Amiga” it would be nice to have a bare-metal version like there is for C64 and 8086 emulation (BMC64, Faux86).

**Comment by h5n1xp** [comment]
It you look at my Omega project I have started such a project, though I decided not use base it on UAE, as I’m not aiming for as high a level of compatibility as UAE does.

Ultimately, my idea is to have just enough emulation to get AmigaOS running, from there I will use AmigaOS drivers to access the RaspberryPi hardware directly.

**Comment by midwan** [comment]
It would be nice, but that's an entirely different project I'm afraid.
Amiberry uses several libraries to get things done, such as SDL2 for graphics, input handling, event handling, sound, etc. (and a few others for special tasks, like mpeg decoding).
For a bare-metal version, all those would have to be implemented as well, somehow (or provided by a bare-metal framework).
